src/js/features/metadata/actions.js

```diff
@@ -26,10 +27,13 @@ export const addOffer = (seller) => ({
     margin: seller.margin,
     arbitrator: seller.arbitrator,
     limitL: seller.useCustomLimits ? seller.limitL.toFixed(2).toString().replace('.', '') : 0,
-    limitU: seller.useCustomLimits ? seller.limitU.toFixed(2).toString().replace('.', '') : 0
+    limitU: seller.useCustomLimits ? seller.limitU.toFixed(2).toString().replace('.', '') : 0,
+    stake: offerStake
```

Comment:
I don't think we should pass the stake in the action, the saga should just know what the value is needed

Reply:
good idea!